### ISSUE TITLE
why do you use scores*20 in UniXcoder

### ISSUE THREAD

**Issue by skye95git** [microsoft/CodeBERT#162]
`loss = loss_fct(scores*20, torch.arange(code_inputs.size(0), device=scores.device))` in `UniXcoder/downstream-tasks/code-search/run.py` 

Why do you use `scores*20` when calculating loss? Like the temperature parameter?

**Comment by skye95git** [comment]
What factors should be referred to to set the temperature parameters?

**Comment by guoday** [comment]
scores are cosine distance, the range of which is [-1,1]. We use `scores*20` to rescale the range to [-20,20] so that the probability can be 0 or 1 after using softmax.
